db/schema/04_messages.sql

```diff
@@ -7,5 +7,5 @@ CREATE TABLE messages (
   receiver_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
   item_id INTEGER REFERENCES items(id) ON DELETE CASCADE,
   created_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
-  body TEXT
+  body TEXT NOT NULL
```

Comment:
Great catch! Do users see a validation message when their message gets rejected because it's empty? That's something we could accomplish with jQuery! Would be happy to pair up for that if you like. :)

Reply:
They don't see anything yet, i would love to pair up!